src/traces/ohlc/transform.js

```diff
@@ -132,6 +133,11 @@ exports.calcTransform = function calcTransform(gd, trace, opts) {
         close = trace.close,
         textIn = trace.text;
 
+    var openName = _(gd, 'Open') + ': ';
+    var highName = _(gd, 'High') + ': ';
+    var lowName = _(gd, 'Low') + ': ';
+    var closeName = _(gd, 'Close') + ': ';
```

Comment:
Should `:` be part of the translation? (Applies a bunch of other places as well)

Reply:
I suppose in principle one could make that argument about other punctuation as well (`,`, `(`, `)`...) but that seems like a can of worms we don't really want to open...

Reply:
> Should : be part of the translation?

I think so. For example, in french it is customary to add a nbsp before `:`, `;`, `!` and `?` e.g. `Open: 10` would be `Ouverture : 10`.

Reply:
moved colons into the translation keys in https://github.com/plotly/plotly.js/pull/2195/commits/bf2c6dbf8619718acd41ff0519f9c00abd0f62fb